test/check-machines-create

```diff
@@ -174,22 +174,24 @@ class TestMachinesCreate(machineslib.VirtualMachinesCase):
                                              is_warning=True)
 
         # user login
+        errors = {"create-vm-dialog-user-login": "User login must not be empty when user password is set"}
         runner.checkDialogFormValidationTest(TestMachinesCreate.VmDialog(self, sourceType='cloud',
                                                                          storage_size=10, storage_size_unit='MiB',
                                                                          location=config.VALID_DISK_IMAGE_PATH,
                                                                          user_password="catsaremybestfr13nds",
                                                                          create_and_run=True),
-                                             {"create-vm-dialog-user-login": "User login must not be empty when user password is set"})
+                                                                        errors)
```

Comment:
This indentation change is confusing -- it now looks like `errors` is an argument to `VmDialaog()`, but it's not -- it's the second argument to `checkDialogFormValidationTest`. Same for the instances below.

Reply:
Good catch.